src/convertToObject.js

```diff
@@ -3,16 +3,28 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const keyAndValueArray = sourceString
+    .split(';')
+    .map(item => item.trim())
+    .filter(item => item !== '');
+
+  const convertedObject = Object.fromEntries(keyAndValueArray
+    .map(element => element.split(':')
+      .map(item => item.trim())
+    ));
+
```

Comment:
Isn't 4 loops too much? I bet you can do everything with either 1 `forEach` or `reduce`